pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/pubsub/MercuryOrderStatusListener.java

```diff
@@ -24,7 +24,7 @@ private MercuryOrderStatusListener(){
         throw new IllegalStateException("Utility class");
     }
 
-    public static void subscribeToOrderStatus(String projectId, String subscriptionId) {
+    public static Subscriber subscribeToOrderStatus(String projectId, String subscriptionId) {
```

Comment:
A few pubsub-related classes were changed so that callers could keep track of (and shutdown) the created subscribers.